src/contributing.md

```diff
@@ -46,13 +46,13 @@ When you push to a pull request, GitHub automatically checks that your changes h
 To run unit tests or Web Platform Tests against a pull request, add one or more of the labels below to your pull request.
 If you do not have permission to add labels to your pull request, add a comment on your bug requesting that they be added.
 
-| Label | Runs unit tests on | Runs web tests on |
-|---|---|---|
-| `T-full` | All platforms | Linux |
-| `T-linux-wpt-2013` | Linux | Linux (only legacy layout) |
-| `T-linux-wpt-2020` | Linux | Linux (skip legacy layout) |
-| `T-macos` | macOS | (none) |
-| `T-windows` | Windows | (none) |
+| Label              | Runs unit tests on | Runs web tests on          |
+|--------------------|--------------------|----------------------------|
+| `T-full`           | All platforms      | Linux                      |
+| `T-linux-wpt-2013` | Linux              | Linux (only legacy layout) |
+| `T-linux-wpt-2020` | Linux              | Linux (skip legacy layout) |
+| `T-macos`          | macOS              | (none)                     |
+| `T-windows`        | Windows            | (none)                     |
```

Comment:
hmm, generally i think we’ll want to prioritise diff and history readability over the readability that comes with aligning things, but let’s go ahead with the change and see how it goes :)

Reply:
Would adding a markdown linter CI workflow be an option, to enforce some style consistency (e.g. denying too long lines, or reminding people to add alt-text for embedded pictures)?

Reply:
maybe! the long lines are intentional, but i would imagine we could configure that kind of thing. i’m not familiar with tooling for markdown, do you have a preferred linter?

Reply:
> the long lines are [intentional](https://book.servo.org/style-guide.html)

I see

> i’m not familiar with tooling for markdown, do you have a preferred linter?

Well, I'm not too familiar either honestly. I mostly use IntelliJ IDEs (RustRover / CLion) when developing, and I couldn't find any detail on if they are using an existing opensource linter, or have a custom linter. 
Some quick googling found [markdownlint](https://github.com/DavidAnson/markdownlint), which seems to be quite configurable and also has an associated github action.

Perhaps we should move further discussion into an issue though, since adding a markdown linter should probably be done in a separate PR and would require further discussion.